src/Contexts/Adaptors/EVMAdaptors.tsx

```diff
@@ -220,7 +220,7 @@ export const EVMHomeAdaptorProvider = ({
         const threshold = BigNumber.from(
```

Comment:
We don't need this code. Let's comment it, add a comment with JIRA ID to ticket in the scope of which we are going to fix it by getting this configuration from the right SC and initiate a new `threshold` variable with `2`.

Reply:
updated.